### DISCUSSION TITLE
Problem on Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by mecrider** [realmazharhussain/gdm-settings#83] [Q&A]
I found this app through https://fostips.com/app-configure-ubuntu-2204-login-screen/. I am working on laptops that originally had 20.04 installed and was upgraded to 22.04. On the previous version I had used the directions at https://tipsonubuntu.com/2020/05/21/hide-user-list-ubuntu-20-04-login-screen/ to disable the user list and used a script at https://github.com/PRATAP-KUMAR/focalgdm3 (no longer there) to add an image background to the login screen. After the upgrade I still had the user list disabled but only had a black background. I tried 2 other scripts that are supposed to set image backgrounds, but both resulted in breaking gdm (Something went wrong...). I downloaded the AppImage of v0.6, and did succeed in setting an image on the first try. However, I lost the disabled user list. I went back to the app and changed settings on Top Bar to use AM/PM time format and on Miscellaneous to disable the user list and add a welcome message. After applying settings and rebooting none of these changes took affect, although the image was still in place. I ran the app again and it still showed all the changes I made, but I still have not seen them on subsequent reboots.

**Comment by realmazharhussain** [comment]
Sorry for being late. I read your question the day you posted it and then forgot about it.

If you still want the issue to get fixed, try running the app inside a terminal with `-v` option (that is, `./gdm-settings.AppImage -v` command). And if it prints any errors (no matter if they occur on hitting 'Apply' or without doing anything at all), you can report back (here or preferably by opening an issue) and I will try and fix it.

Also, try [this AppImage](https://github.com/realmazharhussain/gdm-settings/files/9258463/gdm-settings.AppImage.zip) instead of `v0.6`.

**Reply by mecrider** [reply]
I've tried the new version with -v on 2 computers immediately after upgrading them to 22.04 from 20.04. One had previously had the list of users turned off, which carried through the upgrade, the other was unchanged from a default install. gdm-settings did not give any errors:

> INFO: Application Version    = 0.6.1
> INFO: Operating System       = Ubuntu 22.04.1 LTS
> INFO: PackageType            = AppImage
> INFO: ShellGresourceFile     = /usr/share/gnome-shell/gnome-shell-theme.gresource
> INFO: UbuntuGdmGresourceFile = /usr/share/gnome-shell/gdm-theme.gresource
> INFO: TEMP_DIR               = /root/.cache/app/io.github.realmazharhussain.GdmSettings
> INFO: HOST_DATA_DIRS         = /usr/local/share:/usr/share
> INFO: GDM 42.0
> INFO: pkexec version 0.105
> INFO: GLib 2.72.1
> INFO: Backing up default shell theme …
> INFO: Applying GResource settings for Ubuntu …
After rebooting, the background image I specified was in place, but the user list was no longer hidden on either computer (I marked to hide it in the app), and none of the other changes took affect (adding a welcome message, setting the clock to AM/PM and showing the weekday by the clock).

**Comment by realmazharhussain** [comment]
How about [this AppImage](https://github.com/realmazharhussain/gdm-settings/files/9437587/gdm-settings.AppImage.zip)?
